taf/GitRepository.py

```diff
@@ -24,6 +24,7 @@ def __init__(self, root_dir, target_path=None, repo_urls=None, additional_info=N
       self.repo_path = os.path.join(root_dir, target_path.replace('/', os.sep))
```

Comment:
Maybe we can use `pathlib` like in `repositoriesdb.py` module?

Reply:
I would prefer not to modify target path because it is specified in both `repositories.json` and `mirrors.json` with `/`. I think it's better to keep it consistent with that

Reply:
Sure, but I still think that we can simplify it.

```python
self.target_path =  target_path
self.repo_path = (Path(root_dir) / (target_path or '').replace('/', os.sep)).resolve(True)
```

I think `pathlib` handler os separators, so maybe we can remove `.replace('/', os.sep)`